src/lib.jl

```diff
@@ -516,3 +515,35 @@ Python function.
 function entropic_gromov_wasserstein(μ, ν, Cμ, Cν, ε, loss="square_loss"; kwargs...)
     return pot.gromov.entropic_gromov_wasserstein(Cμ, Cν, μ, ν, loss, ε; kwargs...)
 end
+
+"""
+    mm_unbalanced(a, b, M, reg_m; kwargs...)
+
+Solve the unbalanced optimal transport problem and return the OT plan.
+The function solves the following optimization problem:
+
+```math
+    W = \\min_\\gamma \\quad \\langle \\gamma, \\mathbf{M} \\rangle_F +
+    \\mathrm{reg_{m1}} \\cdot \\mathrm{div}(\\gamma \\mathbf{1}, \\mathbf{a}) +
+    \\mathrm{reg_{m2}} \\cdot \\mathrm{div}(\\gamma^T \\mathbf{1}, \\mathbf{b}) +
+    \\mathrm{reg} \\cdot \\mathrm{div}(\\gamma, \\mathbf{c}) \\\\
+
+    s.t.
+         \\gamma \\geq 0
```

Comment:
Maybe reformulate as e.g.
```suggestion
W = \\min_{\\gamma \\geq 0} \\langle \\gamma, M \\rangle_F +
    \\mathrm{reg_{m1}} \\cdot \\operatorname{div}(\\gamma \\mathbf{1}, a) +
    \\mathrm{reg_{m2}} \\cdot \\operatorname{div}(\\gamma^\\mathsf{T} \\mathbf{1}, b) +
    \\mathrm{reg} \\cdot \\operatorname{div}(\\gamma, c)
```
to be more consistent with other docstrings.